rt/mlc/TOPR2.MLC

```diff
@@ -0,0 +1,1779 @@
+***********************************************************************
+* z390 - Mainframe assembler emulator and run-time engine
```

Comment:
I noticed the running the program creates rt/mlc/TOPR2.TXT
I'm not sure what the best approach is - I think we need to do one of the following:
1. add the file to the .gitignore master file in the root
2. choose an extension that is ignored anyway

Since the file is mechanically produced, I think it should not be included in the repo. Without measures, sooner or later, it will.

Reply:
John changed the file's extension to .TST